src/Diagnostics.DataProviders/ChangeAnalysisClient.cs

```diff
@@ -310,6 +318,46 @@ public async Task<string> PrepareAndSendRequest(string requestUri, object postBo
                 requestMessage.Headers.Add(ClientPrincipalNameHeader, clientPrincipalNameHeader);
             }
 
+            string clientIssuer = string.Empty;
```

Comment:
One way you can improve this is have a string array of all the header names and iterate over this list grabbing the values and adding them to your requestMessage